lib/improver/tests/wind_direction/test_WindDirection.py

```diff
@@ -451,6 +497,15 @@ def test_returns_expected_values(self):
         self.assertArrayAlmostEqual(
             confidence_measure, expected_confidence_measure)
 
+    def test_with_backup(self):
+        """Test raises domain-to-small error when backup method invoked."""
```

Comment:
Is this testing an error raised from the neighbourhooding code?  If so it should be covered in those unit tests and doesn't need to be covered here.

Reply:
It is testing that the backup method is triggered. I'll clarify it a bit...